tests/foreman/api/test_repository.py

```diff
@@ -35,6 +35,9 @@
 from robottelo.api.utils import upload_manifest
 from robottelo.config import settings
 from robottelo.constants import repos as repo_constants
+from robottelo.constants import RPM_TO_UPLOAD
+from robottelo.constants.repos import CUSTOM_FILE_REPO
```

Comment:
Missed this earlier. `from robottelo.constants import repos as repo_constants` is already here, you can already reference this with `repo_constants.CUSTOM_FILE_REPO`. 

`from robottelo import constants` is also already defined above. So you can reference `constants.RPM_TO_UPLOAD`.

Both of these imports are redundant.

Reply:
Noted